### DISCUSSION TITLE
How do you know what version of Sage is in which docker version?

### DISCUSSION THREAD

**Discussion by wayner9** [OpenSageTV/Community#74] [General]
I can see that the SageTV Java11-Opendct dockers are located at https://hub.docker.com/r/sagetvopen/sagetv-server-opendct-java11/tags .  There are versions from 2.01 to 2.07 available.  I believe that these correspond to various versions of SageTV.  How do I know which docker container contains which version of SageTV?

I want to load the docker with SageTV version 9.2.8.1

**Comment by jusjoken** [comment]
Every docker container version will load the latest SageTV version if the
default latest is used
